static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -420,6 +420,12 @@ class GroupHeader extends Component<Props, State> {
                 {t('Similar Issues')}
               </ListLink>
             )}
+            <ListLink
```

Comment:
This and `groupReplays` will need to be feature flagged, see `replays/index` for an example